### PR TITLE
Keep rollout procedure for posterity

### PR DESCRIPTION
**Pull request by rwongone**

### CODE REVIEW

File: scripts/swap_db.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# backup database
+docker run -it --rm --network sys_default postgres:9.4 pg_dump -h db -U postgres testifi_production > dump.sql
+
+# kill app
+make stop
```
Comment (jchorl) [inline scripts/swap_db.sh]: We also need to manually `docker rm` the db now that we took it out of `make stop`

File: scripts/swap_db.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
```
Comment (jchorl) [inline scripts/swap_db.sh]: We also need to `docker volume rm sys_testifi_data` now that the data is stored on a volume

File: scripts/swap_db.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+# start a new database with the volume
+docker run -it --rm -v sys_testifi_data:/var/lib/postgresql/data/testifi -e PGDATA=/var/lib/postgresql/data/testifi -p 5432 postgres:9.4
+
+# restore the database
+docker cp dump.sql 00120a5be610:/dump.sql
```
Comment (jchorl) [inline scripts/swap_db.sh]: This id won't work obv

**Comment by jchorl** [comment]
Good documentation. Some quirks on how this changed today. Also, for dev, the database has a different name (I think testifi_development). While I like having this documented, this should *never* actually be run as a script, it's way too dangerous. I'd probably comment out the whole thing just to make sure. We need to test anything before the chance of losing data. I'm still not 100% certain that my migration today kept _everything_ how it was. e.g. we may find in the future that a primary key counter restarted from 0 (I've had that before but tested for it and it seems fine).

File: scripts/swap_db.sh (hunk 2 above)
Comment (rwongone) [reply scripts/swap_db.sh]: Can this happen after we create and fill the new DB?

**Comment by jchorl** [comment]
Yeah - should be fine I think. Might have port conflicts but I think its
totally fine.

**Pull request merged**